.expeditor/config.yml

```diff
@@ -36,3 +38,23 @@ merge_actions:
         - "Expeditor: Skip Build"
         - "Expeditor: Skip All"
       only_if: built_in:bump_version
+
+# I copied the `ignore_labels` from the chef-dk expeditor config
+# since the labels apply to that workload (not the chef-workstation
+# workload like the `merge_actions` block)
+subscriptions:
```

Comment:
@chef/expeditor-maintainers Does this look like a valid subscription? Because Chef Workstation includes ChefDK I would like to build Chef Workstation on every ChefDK PR merge.

I understand that this means Chef Workstation may get a bad ChefDK because the omnibus build of ChefDK may fail. But we rebuild that whole process in the Chef Workstation pipeline, so if ChefDK fails then Chef Workstation should also fail.

Reply:
Since ChefDK does some post-merge activities, what you probably want to subscribe to is the completion event: https://expeditor.chef.io/docs/reference/workloads/#completed-workloads. So your subscription would look like this: 

```
- workload: chef/chef-dk:master_completed:pull_request_merged:chef/chef-dk:master:*
   ...
```